devoutils/__version__.py

```diff
@@ -1,7 +1,7 @@
 __description__ = 'Devo Python Library.'
 __url__ = 'http://www.devo.com'
-__version__ = "2.0.1"
+__version__ = "2.0.2"
 __author__ = 'Devo'
 __author_email__ = 'support@devo.com'
 __license__ = 'MIT'
-__copyright__ = 'Copyright 2019 Devo'
+__copyright__ = 'Copyright 2020 Devo'
```

Comment:
In devo we maintain the copyright from the year of the first version release, Gonzalo told me that it is correct to leave it in 2019

Reply:
Ok I'll revert the change